### PR TITLE
Billing - UI - consume new REST endpoints

### PR DESCRIPTION
**Pull request by ClaudeROSSI**
Operations targeting the payment methods are now targeting the new REST endpoints

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,6 +110,7 @@
     "socket.io-client": "^3.1.3",
     "sweetalert2": "^10.15.6",
     "tslib": "^2.1.0",
+    "url-pattern": "^1.0.3",
```
Comment (jerome-benoit) [inline package.json:113]: I would prefer that you use a widely used URL parser like parseurl. I would even prefer that you rely only on express URI parameters extraction but that seems difficult here. 

You can use https://www.npmtrends.com/url-pattern-vs-parseurl-vs-parseuri-vs-url-parse to compare npm package usage.

Comment (ClaudeROSSI) [reply package.json:113]: thanks for the hints ... i'll have a look. we also need to escape url fragments. Right now this is not done.

Comment (ClaudeROSSI) [reply package.json:113]: > I would prefer that you use a widely used URL parser like parseurl.
> 
> You can use https://www.npmtrends.com/url-pattern-vs-parseurl-vs-parseuri-vs-url-parse to compare npm package usage.

Well ... I do not need an URL parser. I simply need an URL builder.
The one matching my needs is: https://www.npmjs.com/package/safe-url-assembler
Are you aware of an alternative? How to generate URLs matching endpoints defined using dynamic express routes such as: /things/:id/:name

Comment (jerome-benoit) [reply package.json]: No but go for that one then.

File: src/app/services/central-server.service.ts [hunk 2]
```diff
@@ -3274,4 +3287,15 @@ export class CentralServerService {
     }
     return of(null);
   }
+
+  private buildRestEndpointUrl(urlPatternAsString: ServerRoute, params: {
+    // Just a flat list of key/value pairs!
+    [name: string]: string | number | null;
+  }) {
+    const url = SafeUrlAssembler(this.restServerSecuredURL)
+      .template('/'+urlPatternAsString)
```
Comment (jerome-benoit) [inline src/app/services/central-server.service.ts:3296]: nitpick: space before and after `+`.

**Review by LucasBrazi06 — APPROVED** [review]
Check Jerome's comment.
Mergeable anyway :-)

**Pull request merged**